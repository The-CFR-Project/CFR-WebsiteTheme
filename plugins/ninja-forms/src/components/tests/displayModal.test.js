import { render, cleanup } from '@testing-library/react';
import { DisplayModal } from '..';

describe( 'submissions-display-modal', () => {
	afterEach( cleanup );

	const closeComp = jest.fn();
	const setModalOpen = jest.fn();
	it( 'Matches snapshot', () => {
		const { container } = render(
			<DisplayModal
				title="Cancel Bulk Export?"
				actionText="Yes"
				action={ closeComp }
				cancelText="No"
				cancel={ setModalOpen }
			/>
		);
		expect( container ).toMatchSnapshot();
	} );
} );
