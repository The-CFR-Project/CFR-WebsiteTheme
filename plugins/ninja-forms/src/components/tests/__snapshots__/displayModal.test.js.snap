// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-display-modal Matches snapshot 1`] = `
<div
  aria-hidden="true"
/>
`;
